### ISSUE TITLE
[1.17] /version doesn't output anything

### ISSUE THREAD

**Issue by sehrschlechtYT** [PaperMC/Paper#5880]
### Expected behavior

It shows "This server is running Paper version git-Paper-"d76f8e0" (MC: 1.17) (Implementing API version 1.17-R0.1-SNAPSHOT)"

### Observed/Actual behavior

It shows _Checking version, please wait_. After that, it doesn't even show this message anymore when executing the command again.

### Steps/models to reproduce

1. Start a Paper 1.17 Server (3rd Experimental Build)
2. Execute /version

### Plugin list

- ProtocolLib

### Paper version

This server is running Paper version git-Paper-"d76f8e0" (MC: 1.17) (Implementing API version 1.17-R0.1-SNAPSHOT)

### Agreements

- [X] I am running the latest version of Paper available from https://papermc.io/downloads.
- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [X] My version of Minecraft is supported by Paper.

### Other

The server shows the version on startup, but not with the command.

**Comment by darbyjack** [comment]
This is most likely because there's no API for 1.17 published yet.

**Comment by sehrschlechtYT** [comment]
But why does it show the message "This server is running Paper version git-Paper-"d76f8e0" (MC: 1.17) (Implementing API version 1.17-R0.1-SNAPSHOT)" on startup then?

**Comment by Machine-Maker** [comment]
Because it knows the version the server is running since it’s itself.

**Comment by Chew** [comment]
Because that's the version. git-Paper-buildNumber. Usually, it's a build number from the API so it uses that, but if it's self-compiled (as the test builds are), it'll be the git hash. Paper can't check the version because 1.17 isn't released on the API yet, so there's no "build" to check against.
tl;dr it does show the version, just not how outdated you are, because there's no way to check

**Comment by BillyGalbreath** [comment]
I think this should be considered a valid bug report since it no longer says "Unknown version" after checking history.

For example, I still use the old version checker in my fork (the one that checks jenkins) and this is the output I get

![image](https://user-images.githubusercontent.com/332527/122650071-73e74280-d0f6-11eb-84d1-2ab547c26002.png)

This bug report is claiming that Paper just doesn't output anything at all anymore after saying its checking the history.

**Comment by Chew** [comment]
That is fair (and I can reproduce), can someone update the title, then?
Very minor issue, though.

**Comment by electronicboy** [comment]
No response in almost a week and last time I saw this it was down to a 3rd party client
